management/branding.go

```diff
@@ -116,6 +116,10 @@ type BrandingFont struct {
 	URL *string `json:"url,omitempty"`
 }
 
+type BrandingTemplateUniversalLogin struct {
```

Comment:
@alexkappa I am wondering if I need to create a new standalone resource or if I follow the [documentation](https://auth0.com/docs/api/management/v2#!/Branding/get_universal_login) and add it like it is right now?

Reply:
I saw that you did already the same in user for example